### ISSUE TITLE
Revise dependency bounds

### ISSUE THREAD

**Issue by Bodigrim** [channable/alfred-margaret#62]
Thanks for this package, I've been very much impressed by its performance! Could you please revise upper bounds for dependencies on Hackage? No code changes are needed.

```
$ cabal outdated
Outdated dependencies:
containers >=0.6 && <0.7 (latest: 0.7)
deepseq >=1.4 && <1.5 (latest: 1.5.0.0)
hashable >=1.4.0.2 && <1.5 (latest: 1.5.0.0)
primitive >=0.6.4 && <0.8 (latest: 0.9.0.0)
text >=2.0 && <2.1 (latest: 2.1.1)
vector >=0.12 && <0.13 (latest: 0.13.1.0)
```

**Comment by ReinierMaas** [comment]
Hey @Bodigrim,

Thanks for the request. We have updated the package to support the new bounds and published `v2.1.0.2` to Hackage today.

**Comment by Bodigrim** [comment]
Thanks, awesome!
